acme/acme/client.py

```diff
@@ -603,10 +603,8 @@ def query_registration(self, regr):
             Resource.
 
         """
-        self.net.account = regr
```

Comment:
Looks like this line is here for a reason: https://github.com/certbot/certbot/issues/6258

Reply:
Added the line back.